dom/events/Event-dispatch-on-disabled-elements.html

```diff
@@ -98,6 +98,123 @@
   }
 }, "Calling click() on disabled elements must not dispatch events.");
 
+promise_test(async () => {
+  // Based on GlobalEventHandlers and mixins
+  // https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers
+  // https://drafts.csswg.org/css-transitions/#interface-globaleventhandlers-idl
+  const globalEventHandlers = new Map([
+    [
+      DragEvent,
+      [
+        "drag",
+        "dragend",
+        "dragenter",
+        "dragexit",
+        "dragleave",
+        "dragover",
+        "dragstart",
+        "drop",
+      ],
+    ],
+    [
+      Event,
+      [
+        "abort",
+        "cancel",
+        "canplay",
+        "canplaythrough",
+        "change",
+        "close",
+        "cuechange",
+        "durationchange",
+        "emptied",
+        "ended",
+        "input",
+        "invalid",
+        "loadeddata",
+        "loadedmetadata",
+        "loadend",
+        "pause",
+        "play",
+        "playing",
+        "progress",
+        "ratechange",
+        "reset",
+        "securitypolicyviolation",
+        "seeked",
+        "seeking",
+        "select",
+        "stalled",
+        "submit",
+        "suspend",
+        "timeupdate",
+        "toggle",
+        "volumechange",
+        "waiting",
+      ],
+    ],
+    [FormDataEvent, ["formdata"]],
+    [KeyboardEvent, ["keydown", "keypress", "keyup"]],
+    [
+      MouseEvent,
```

Comment:
Actually, some of these mouse ones shouldn't be here... the ones that map to a "click".